booking-up-for-beauty/jabieren/booking_up_for_beauty.go

```diff
@@ -0,0 +1,39 @@
+package booking
+
+import (
+       "time"
+)
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	layout := "1/02/2006 15:04:05"
```

Comment:
Since you can't count on there being double digits in the day of the month, you should not use the leading 0:
```
layout := "1/2/2006 15:04:05"
```

Reply:
Actually you are right. I did the same assuming that it was a double digit, but nothing in the exercise says so.